client/src/components/Feed/PostCard.js

```diff
@@ -1,22 +1,36 @@
 import { Card } from "antd-mobile";
 import styled from "styled-components";
-import { theme } from "constants/theme";
+import { theme, mq } from "constants/theme";
 
 const { colors, typography } = theme;
 const { royalBlue, darkGray } = colors;
 const { display } = typography.font.family;
-const { xsmall, medium, large, xxlarge } = typography.size;
+const { xsmall, small, medium, large, xxlarge } = typography.size;
 
 const PostCard = styled(Card)`
```

Comment:
- There are a few border lines that appear at smaller screen sizes which shouldn't appear
- The post description needs less margin-left or padding-left at smaller screen sizes

current
<img width="316" alt="Screen Shot 2020-05-10 at 7 53 43 PM" src="https://user-images.githubusercontent.com/42228386/81521139-3745f680-92fb-11ea-9bc8-b684f338f4ff.png">

figma
<img width="457" alt="Screen Shot 2020-05-10 at 7 53 36 PM" src="https://user-images.githubusercontent.com/42228386/81521237-72e0c080-92fb-11ea-960c-766a2fbb0fb5.png">

Reply:
- Posts need a border in Desktop UI
- Tags needs to be next to the post title (not under)
- Need the three dots at the right corner of the post
- Font sizes of the post are slightly bigger 

current
<img width="1014" alt="Screen Shot 2020-05-10 at 8 22 36 PM" src="https://user-images.githubusercontent.com/42228386/81521403-fef2e800-92fb-11ea-8fd0-4079fa762de6.png">

figma
<img width="915" alt="Screen Shot 2020-05-10 at 8 21 19 PM" src="https://user-images.githubusercontent.com/42228386/81521408-031f0580-92fc-11ea-9f07-7f7496f368b3.png">

Reply:
Done.
Reviewed font sizes and they are the same, maybe description in figma uses DM Sans, no Poppins. Please review.

Reply:
Awesome, I think you need to push your changes? Don't see theme.